manager/assets/modext/sections/resource/update.js

```diff
@@ -103,27 +103,33 @@ Ext.extend(MODx.page.UpdateResource,MODx.Component,{
 
     ,getButtons: function(cfg) {
         var btns = [];
-        if (cfg.canSave == 1) {
+        
+        //if (cfg.locked) {
+            btns.push({
+                text: cfg.lockedText || _('locked')
+                ,id: 'modx-abtn-locked'
+                ,handler: Ext.emptyFn
+                ,hidden: (cfg.canSave == 1)
```

Comment:
Before the check was on cfg.locked - I understand the logic of rendering the buttons despite it being locked or not, but are you sure this check is correct?